test/suites/snapshots.sh

```diff
@@ -440,9 +453,9 @@ test_snap_volume_db_recovery() {
   lxc snapshot c1
   lxc start c1
   lxc stop -f c1
-  lxd sql global 'DELETE FROM storage_volumes_snapshots' # Remove volume snapshot DB records.
+  lxd sql global 'DELETE FROM storage_volumes_snapshots'                               # Remove volume snapshot DB records.
```

Comment:
looks like your editor is changing things on save, but these are not related to the test commit:

> test/suites/snapshot: Add test to preserve definition in snap_restore 

So they should either be reverted or added in a separate commit.